.github/workflows/ci.yml

```diff
@@ -957,6 +957,7 @@ jobs:
     name: Check governance
     runs-on: ubuntu-latest
     needs: 'genmatrix'
+    if: ${{ github.event_name == 'push' || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' }}
```

Comment:
I think this may need to be moved back to where it was before

Reason being is that there's a `needs: [tests, checkgovernance]` check lower down which blocks the job from runing ... and if checkgovernance never runs that may cause an issue?

Reply:
"where it was before" doesn't exist anymore. I've added these conditions also to `dispatchrelease` to avoid any possible issues.
Check governance should never have been done on pull requests in the first place, since it's only needed if we're tagging stuff and we don't want to tag a PR.

Reply:
Seems like it should also be removed from here though, because we have:

`needs: [tests, checkgovernance]` in `dispatchtagpatchrelase`

My concern is that if `checkgovernance` doesn't run then neither will `dispatchtagpatchrelase` - possibly it will, I'm just not sure how github actions will react in that scenario.

Reply:
If `checkgovernance` doesn't run, it'll be because of the same condition that was added to `dispatchtagpatchrelase`, so they both fail the `if` condition. In any case, I'll run it in a repo I control so we can make an informed decision.

Reply:
https://github.com/GuySartorelli/silverstripe-composable-validators/actions/runs/10188308891/job/28326019758?pr=9
Both jobs get correctly skipped. No need to change it.